### PR TITLE
Website update

### PR DESCRIPTION
**Pull request by SerSquirt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,15 +6,24 @@
 	<meta name="viewport" content="width=device-width">
 	<title>SerSquirt's Website</title>
 	<link href="style.css" rel="stylesheet" type="text/css" />
+  <html lang="en-US">
```
Comment (MarkSort) [inline index.html:9]: This is an extra `html` tag.  There is also one on line 2.  But the one on line 2 should have `lang="en-US"` like this one.

Comment (SerSquirt) [reply index.html]: Fixed!

**Review by MarkSort — APPROVED** [review]
Awesome! LGTM!

**Comment by SerSquirt** [comment]
It's merged! 😁

**Pull request merged**